libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -18,12 +17,26 @@ static char WGS_84_FRAME_ID[] = "WGS-84";
 // https://www.ros.org/reps/rep-0105.html#base-link
 static char BASE_LINK_FRAME_ID[] = "base_link";
 
-AP_HAL::UARTDriver *dds_port;
-
-
-const AP_Param::GroupInfo AP_DDS_Client::var_info[]= {
-    //! @todo Params go here
-
+const AP_Param::GroupInfo AP_DDS_Client::var_info[] {
+
+    // @Param: _ENABLED
+    // @DisplayName: DDS enable
+    // @Description: Enable DDS subsystem
+    // @Values: 0:Disabled,1:Enabled
+    // @RebootRequired: True
+    // @User: Advanced
+    AP_GROUPINFO_FLAGS("_ENABLED", 1, AP_DDS_Client, enabled, 0, AP_PARAM_FLAG_ENABLE),
```

Comment:
Please document the params in the README.

Reply:
Use `DDS_ENABLE` instead of `DDS_ENABLED`? Most of the other enable params except chute use the former.

Reply:
done